packages/design-system/src/components/FormLabel/useFormLabel.tsx

```diff
@@ -120,22 +147,19 @@ export function useFormLabel<T extends UseFormLabelProps>(props: T) {
   const fieldProps = {
     ...remainingProps,
     id,
-    errorId,
     errorMessage,
-    errorPlacement,
     inversed,
+    'aria-describedby': hasBottomError ? classNames(props['aria-describedby'], errorId) : undefined,
```

Comment:
These snapshot changes where the `aria-describedby` disappears have been troubling me, and it's a problem with my initial attempt to convert dropdown. I've made the following change locally and updated snapshots, and I think it does a better job of maintaining the old behavior than my previous version:

```suggestion
    'aria-describedby': hasBottomError ? classNames(props['aria-describedby'], errorId) : props['aria-describedby']
```